README.md

```diff
@@ -256,13 +273,16 @@ const { webpackConfig } = require('@rails/webpacker')
 
 console.log(webpackConfig.output_path)
 console.log(webpackConfig.source_path)
+
+// Or to print out your whole webpack configuration
+console.log(JSON.stringify(webpackConfig, undefined, 2)
```

Comment:
There is a typo here. It should be `console.log(JSON.stringify(webpackConfig, undefined, 2))` (missing closing bracket)

Reply:
@n-rodriguez Thanks! Fixed!